Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.4-cli
+FROM php:8-cli
```

Comment:
tady `8.0`, protoze to je minimalni verze v `composer.json`.. takle to vypada, ze podporujes 8.0, ale pritom vyvijis na 8.2